entries/QUnit.stack.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.stack">
+	<title>QUnit.stack()</title>
+	<signature>
+		<argument name="offset" type="Number" optional="true" default="2">
+			<desc>Set the stackTrace line offset.</desc>
+		</argument>
+	</signature>
+	<desc>Return a single line source reference from the stacktrace (call stack)</desc>
```

Comment:
I think that last part should be one word, "callstack". Though this sentence is rather confusing, the first paragraph of the longdesc below is much better.